update.py

```diff
@@ -33,7 +33,7 @@ def msg(message: str) -> None:
         subprocess.check_call(
             [
                 os.path.join(os.getcwd(), ".venv", "bin", "python"),
-                os.path.join(os.getcwd(), "bubblesRTM.py"),
+                os.path.join(os.getcwd(), "bubbles/bubblesRTM.py"),
```

Comment:
I think this should be `bubbles/main.py`

Reply:
That's a great catch -- I haven't updated any of the update / deploy code yet, but this is an easy one to fix.